src/disco/routes.js

```diff
@@ -8,7 +8,7 @@ export default (
   <Route path="/" component={App}>
     <IndexRoute component={DiscoPane} />
     <Route
-      path="/:locale/firefox/discovery/pane/:version/:platform/:compatibilityMode"
+      path="/:lang/firefox/discovery/pane/:version/:platform/:compatibilityMode"
```

Comment:
Throughout all the i18n scripts a lang is formatted consistently as `en-US` and a locale is `en_US`.

The lang format is what is used for the lang attribute. The locale format is used for translations and the directories of translations. All the utils provide can convert from lang to locale.

Since the url input is hyphenated it's called a lang in this case.